build/assets/style.csl

```diff
@@ -50,26 +50,28 @@
   <bibliography hanging-indent="true" second-field-align="margin">
     <layout>
       <text variable="citation-number" suffix=". "/>
-      <group>
-        <text variable="title" font-weight="bold"/>
-      </group>
-      <group display="block">
-        <text macro="author"/>
-      </group>
       <group delimiter=" ">
```

Comment:
The diff here makes 7335415d96f5adc19fae0d03552a906f960468b9 look larger than it actually is.

@adam3smith is the only change that everything has been wrapped in `<group delimiter=" ">...</group>`?